connectors/presto/presto-snowflake/src/test/java/com/facebook/presto/plugin/snowflake/TestingSnowflakeServer.java

```diff
@@ -0,0 +1,139 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.snowflake;
+
+import com.facebook.airlift.log.Logger;
+
+import java.security.SecureRandom;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static java.lang.Thread.sleep;
+import static java.util.Objects.requireNonNull;
+
+public class TestingSnowflakeServer
+{
+    private static final Logger logger = Logger.get(TestingSnowflakeServer.class);
+    private static final char[] ALPHABET = new char[36];
+
+    static {
+        for (int digit = 0; digit < 10; digit++) {
+            ALPHABET[digit] = (char) ('0' + digit);
+        }
+
+        for (int letter = 0; letter < 26; letter++) {
+            ALPHABET[10 + letter] = (char) ('a' + letter);
+        }
+    }
+    private static final int RANDOM_SUFFIX_LENGTH = 10;
+    private static final SecureRandom random = new SecureRandom();
+    public static final String TEST_URL = requireNonNull(System.getProperty("snowflake.test.server.url"), "snowflake.test.server.url is not set");
+    public static final String TEST_USER = requireNonNull(System.getProperty("snowflake.test.server.user"), "snowflake.test.server.user is not set");
+    public static final String TEST_PASSWORD = requireNonNull(System.getProperty("snowflake.test.server.password"), "snowflake.test.server.password is not set");
+    public static final String TEST_DATABASE = requireNonNull(System.getProperty("snowflake.test.server.database"), "snowflake.test.server.database is not set");
+    public static final String TEST_ROLE = requireNonNull(System.getProperty("snowflake.test.server.role"), "snowflake.test.server.role is not set");
+    public static final String TEST_SCHEMA = requireNonNull(System.getProperty("snowflake.test.server.schema"), "snowflake.test.server.schema is not set");
+
+    private TestingSnowflakeServer() {}
+
+    public static void execute(String sql)
+    {
+        execute(TEST_URL, getProperties(), sql);
+    }
+
+    static void execute(String url, Properties properties, String sql)
+    {
+        try (Connection connection = getConnection(url, properties);
+                Statement statement = connection.createStatement()) {
+            statement.execute(sql);
+        }
+        catch (SQLException | InterruptedException e) {
+            e.printStackTrace(System.err);
+            if (e.getCause() != null) {
+                e.getCause().printStackTrace(System.err);
+            }
+            throw new RuntimeException(e);
```

Comment:
Please just bubble up this exception to the top level, don't wrap it, and don't print to console.